content/src/main/content/jcr_root/apps/dam/gui/content/assets/metadataeditor/.content.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same as before; how can we make this feature opt-in?

Reply:
I have added a feature flag to show/hide this button.